go/kbnm/findbin.go

```diff
@@ -11,25 +11,25 @@ import (
 
 var errKeybaseNotFound = errors.New("failed to find the keybase binary")
 
-// findKeybaseBinary returns the path to the Keybase binary, if it finds it.
-func findKeybaseBinary() (string, error) {
+// findKeybaseBinary returns the path to a Keybase binary, if it finds it.
+func findKeybaseBinary(name string) (string, error) {
```

Comment:
There is already GetRuntimeDir() in the global context, which calls some libkb helper functions to get the binary